lxd/daemon.go

```diff
@@ -1307,7 +1302,6 @@ func (d *Daemon) init() error {
 
 	instancePlacementScriptlet := d.globalConfig.InstancesPlacementScriptlet()
 
-	d.endpoints.NetworkUpdateTrustedProxy(d.globalConfig.HTTPSTrustedProxy())
```

Comment:
why did this line move btw?

Reply:
I found that if this is executed before `endpoints.Up()`, we are panicking.

Reply:
`endpoints` need to be up if we want to update I guess

Reply:
What was the stack trace of the panic?

Reply:
I see makes sense :)